public/app/features/dashboard/components/SaveDashboard/SaveDashboardModalProxy.tsx

```diff
@@ -15,6 +17,11 @@ export const SaveDashboardModalProxy: React.FC<SaveDashboardModalProps> = ({ das
     onSaveSuccess,
   };
 
+  // Feature flag to show save as a drawer (and diff) rather than just simple modal
+  if (config.featureToggles.saveWithDiff) {
```

Comment:
We may want to include similar checks as below to determine if we need to display save modal at all (i.e. `isChanged && !isProvisioned` checks)

Reply:
Ideally we can just remove this Proxy and include the `<SaveDashboardDrawer ...` directly.  Internally it supports all these flavors directly